rundoctests.py

```diff
@@ -24,47 +65,55 @@
 for typ in [AlarmInterrupt, SignalError]:
     typ.__module__ = "__main__"
 
+if os.name != 'nt':
+    # Limit stack size to avoid errors in stack overflow doctest
+    stacksize = 1 << 20
+    resource.setrlimit(resource.RLIMIT_STACK, (stacksize, stacksize))
+
+    # Disable core dumps
+    resource.setrlimit(resource.RLIMIT_CORE, (0, 0))
+
+def testfile(file):
+    # Child process
+    try:
+        if sys.platform == 'darwin':
+            from cysignals.signals import _setup_alt_stack
+            _setup_alt_stack()
+        failures, _ = doctest.testfile(file, module_relative=False, optionflags=flags, parser=parser)
+        if not failures:
+            os._exit(0)
+    finally:
+        os._exit(23)
+
+if __name__ == "__main__": # Mandatory for windows cases.
```

Comment:
`windows cases` sounds strange to me. If you do put a comment, I would prefer `# Needed for Windows` or something like that. Alternatively: this construction is standard enough, I don't think that it needs a comment.